cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/aggregation/NGSIGenericColumnAggregator.java

```diff
@@ -128,6 +130,31 @@ public void aggregate(NGSIEvent event) {
                 aggregation.put(attrName + "_type", valuesType);
             } // if else
         } // for
+
+        if (event.getMappedCE() != null && isEnableNameMappings() && isEnableGeoParse()) {
```

Comment:
We do this Just when isEnableNameMappings? I can't find where we do this for the base case.